content/millennium/dstu2/general-clinical/procedure.md

```diff
@@ -36,11 +56,25 @@ Notes:
 
   - The `date` parameter must have a time, may be provided up to two times, and must use the `ge` or `le` prefixes. When provided twice, the lower value must have the `ge` prefix and the higher value must have the `le` prefix.
 
-### Response
+### Headers
+
+ <%= headers %>
+
+### Example
```

Comment:
This is outside the scope of this code review but I just wanted to note that the header sizes for Example and Request/Response look exactly the same in my browser. Request/Response don't generate a TOC link, but it looks slightly odd to me.